packages/utils/ast-utils.ts

```diff
@@ -1,6 +1,24 @@
 import { JSCodeshift, Node, valueType } from "./types/NodePath";
 import * as validateIdentifier from "./validate-identifier";
 
+function isImportPresent(j: JSCodeshift, ast: Node, path: string): boolean {
+	if (typeof path !== "string") {
+		throw new Error(`path parameter should be string, recieved ${typeof path}`);
+	}
+	let isPresent = false;
```

Comment:
`mergeImportExists`

Reply:
It is a general utility function, can be used for `topScope` too while adding new packages. Are you okay with "importExists"?